run_contest.py

```diff
@@ -40,7 +40,7 @@ def example_team_reward_func(car):
     return 1
 
 # and then add it to your config like this:
-example_team["reward"]["function"] = example_team_reward_func
+crepe["reward"]["function"] = example_team_reward_func
```

Comment:
If you plan on using the default reward function please delete this line. This section is only for if you wanted to use a custom one